easy_my_coop_website/views/subscription_template.xml

```diff
@@ -1,131 +1,496 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <template id="assets_frontend" inherit_id="web.assets_frontend" name="Coop">
-        <xpath expr="." position="inside">
-            <script
-                type="text/javascript"
-                src="/easy_my_coop/static/src/js/easy_my_coop.js"
+
+    <template id="contact_template" name="contact_template">
+
+
+
+                                <div class="row">
+                                <div class="col-md-6">
+                                <div
+                    class="form-group field-firstname"
+                    t-attf-class="form-group #{error and 'firstname' in error and 'has-error' or ''}"
+                    name="firstname_container"
+                >
+                                    <label for="firstname">Firstname</label>
+                                    <input
+                        class="form-control form-control-sm"
+                        id="firstname"
+                        type="text"
+                        name="firstname"
+                        required="True"
+                        t-att-readonly="logged"
+                        t-attf-value="#{firstname or ''}"
+                        placeholder="Didier"
+                    />
+                                </div>
+                                </div>
+
+                                <div class="col-md-6">
+                                <div
+                    class="form-group field-lastname"
+                    t-attf-class="form-group #{error and 'lastname' in error and 'has-error' or ''}"
+                    name="lastname_container"
+                >
+                                    <label for="lastname">Lastname</label>
```

Comment:
Fixed for last name.
Ah yes I think using name instead of first-last name needs too much work, considering the amount of work that this PR demanded : https://github.com/coopiteasy/vertical-cooperative/pull/177